src/cicero/dom.cto

```diff
@@ -62,6 +62,7 @@ asset Parameter extends Element {
   o String name // the name of the parameter
   o String description optional
   o String type optional // the type of the parameter if defined inline. If not specified the parameter must be in the template model.
+  o ParameterExpression expr optional // An expression to constrain the parameter. Should return a boolean
```

Comment:
How would this relate to a validator expression in the CTO model? `ParameterConstraint`?

Reply:
This would behave similarly as a validator expression in the CTO model, but you could write an Ergo expression to use as a constraint inside your contract (e.g., `{[ paymentDate : DateTime | after(paymentDate,executionDate) ]}`

Reply:
`ConstraintExpression` instead of `ParameterExpression`

Reply:
```
{{ import org.accordproject.money.* }
This is a contract where you have to pay {[ amount : USD ]}
```